modules/service_container_doctrine/tests/modules/service_container_doctrine_test/service_container_doctrine_test.services.yml

```diff
@@ -0,0 +1,2 @@
+parameters:
+  annotated_plugins_auto_discovery: 'service_container_doctrine_test'
```

Comment:
No we will use parameters here:

- [ owner: 'block', directory: 'Plugin/Block' ]